src/app/public/modules/split-view/split-view-workspace-header.component.scss

```diff
@@ -0,0 +1,14 @@
+@import "~@skyux/theme/scss/mixins";
+@import "~@skyux/theme/scss/_compat/mixins";
```

Comment:
I thought that the main mixin import imported the compat one by default. Does it not?

Reply:
It does not.

Reply:
Ok